tests/Concerns/WithGroupedHeadersTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Maatwebsite\Excel\Tests\Concerns;
+
+use Illuminate\Support\Collection;
+use Maatwebsite\Excel\Concerns\Importable;
+use Maatwebsite\Excel\Concerns\ToArray;
+use Maatwebsite\Excel\Concerns\ToCollection;
+use Maatwebsite\Excel\Concerns\WithGroupedHeaders;
+use Maatwebsite\Excel\Concerns\WithHeadingRow;
+use Maatwebsite\Excel\Tests\TestCase;
+use PHPUnit\Framework\Assert;
+
+class WithGroupedHeadersTest extends TestCase
```

Comment:
Can you also add a test for ToModel and OnEachRow?

Reply:
I was originally going to add a ToModel test but for me to do so I believe I will have to add some migrations to allow for an array casted column on the User model. So I held off on that since it would modify much more than my intended targeted slice. Any thoughts on how I should proceed?

Reply:
Just `implode()` the value and store it like that ?

A test with validation would indeed also be really good to have!